source/extensions/extensions_metadata.yaml

```diff
@@ -592,8 +592,8 @@ envoy.stat_sinks.wasm:
 envoy.tls.cert_validator.spiffe:
   categories:
   - envoy.tls.cert_validator
-  security_posture: unknown
-  status: wip
+  security_posture: robust_to_untrusted_downstream_and_upstream
```

Comment:
Do we have a fuzzer for this? I'm not confident at a glance that the error checking from BoringSSL is robust and I think for something that is dealing with security sensitive cert validation a fuzzer would be best.

Reply:
we don't have fuzzer on this, so I will put `requires_trusted_downstream_and_upstream` for now, and after having fuzzer, I will raise the PR again to change to `robust_to_untrusted_downstream_and_upstream`. sg?

Reply:
Sure